project/scripts/main.py

```diff
@@ -17,6 +17,7 @@
 
```

Comment:
This file is now no longer the same as what we have in Cloud Functions.
Also we should move the Cloud Functions scripts into a `functions` directory so it's not confusing when the cronjob scripts are created too.

Reply:
The cloud functions script and the cronjob script use the exact same functions, the only thing that changes is the entry point. Where would I put the generic files used by both the cronjob and the cloud function? (dates, fetch_trends, database_updates etc). Also, I think I copied the cloud function over exactly into the get_context script, main is the cronjob entry point instead. They're slightly different.

Reply:
Good point. Maybe have the generic files in the `scripts` directory, imported by both `functions` and `cronjobs`?
Would be worth keeping the cloud function entry point in git, as something like `functions/main.py`